Sources/ApiVideoPlayer/Views/SelectableListView.swift

```diff
@@ -0,0 +1,129 @@
+#if !os(macOS)
+import AVFoundation
+import Foundation
+import UIKit
+
+class SelectableListView<T: Equatable>: UIView, UITableViewDataSource, UITableViewDelegate {
+    private var tableview = UITableView()
+    private let cellReuseIdentifier = "CellWithTextField"
+
+    private let listAny: [T]
+    private var selectedAny: T?
+
+    public weak var delegate: SelectableViewDelegate?
+
+    public init(frame: CGRect, list: [T], selectedElement: T?) {
```

Comment:
Is there any reason to call input parameter `list` and `selectedElement` whereas their name are `listAny` and `selectedAny` internally?